app/views/application/contact-details/select-address.njk

```diff
@@ -1,5 +1,4 @@
 {% extends "_form.njk" %}
-
```

Comment:
Stylistically, have been separating `extends` from the `set` statements that follow. Can we keep that?

Reply:
Updated